sql-plugin/src/main/301until330-all/scala/com/nvidia/spark/rapids/shims/v2/Spark30Xuntil33XShims.scala

```diff
@@ -28,3 +29,8 @@ trait Spark30Xuntil33XShims extends Spark30Xuntil33XFileOptionsShims {
     GpuOverrides.neverReplaceExec[ShowCurrentNamespaceExec]("Namespace metadata operation")
   }
 }
+
+trait GpuDeterministicAggregateShim extends Expression {
```

Comment:
Aggregate sounds too broad. Maybe `GpuDeterministicFirstLastCollectShim` and let us pull the comment above the trait.